### PR TITLE
fix-call-me

### PR DESCRIPTION
**Pull request by MarinaPasternak**

### CODE REVIEW

File: app/components/callMe/call-controller.js [hunk 1]
```diff
@@ -15,6 +15,6 @@ export class CallController {
 
   handleFormSubmit = (phoneNumber) => {
     const result = this.model.validation(phoneNumber);
-    // передать данные в функцию Вовы через свою модель
+    this.model.sendData(phoneNumber);
```
Comment (PavelKundenko) [inline app/components/callMe/call-controller.js]: Add window closing on submit
(Create such method in view and use this method)

**Review by PavelKundenko — CHANGES_REQUESTED** [review]

File: app/components/callMe/call-controller.js (hunk 1 above)
Comment (MarinaPasternak) [reply app/components/callMe/call-controller.js]: Done, check please it in the View

**Review by MozdolevskyiMaksym — APPROVED** [review]

**Review by OlehDrozdov — APPROVED** [review]

File: app/components/callMe/call-view.js [hunk 2]
```diff
@@ -36,4 +36,16 @@ export class CallView {
     handleChange = (ev) => {
       this.inputValue = ev.target.value;
     };
+
+    closeForm = (ev) => {
+      const form = document.querySelector('.form-show');
+      alert("we will write to you in Telegram within 10 minutes");
+      form.classlist.add(“form-hide”);
```
Comment (PavelKundenko) [inline app/components/callMe/call-view.js]: Such strange quotes.. Does it work?

Comment (MarinaPasternak) [reply app/components/callMe/call-view.js]: Max said  that I should write like that

**Review by rearchontis — APPROVED** [review]

**Review by PavelKundenko — APPROVED** [review]

**Pull request merged**